AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -34,6 +34,10 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 !
 ! SECTION: Subscriptions - Regular rules
 !
+pbteen.com,potterybarn.com,potterybarnkids.com,westelm.com,williams-sonoma.com#%#//scriptlet('set-constant', 'WSI.contentPersonalization.hideEmailCaptureOverlay', 'true')
+markandgraham.com###join-email-campaign
+markandgraham.com#$#body { overflow: auto !important; }
```

Comment:
If scrolling is locked, popup also must be hidden using CSS rule to avoid breakage with Content Blocker for Android.

Reply:
Okay. I updated my PR.